### PR TITLE
add the little big event #1

### PR DESCRIPTION
**Pull request by krichprollsch**
screenshot from 2019-03-05 09-03-29

### CODE REVIEW

File: _posts/2019-03-21-little-big-event-1-typescript.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+title: Little Big Event#1 Typescript
+---
+
+Le jeudi 21 mars 2019, [Mug in Clermont](https://planet.clermontech.org/tags/mug-in-clermont/),
+le [LavaJUG](https://planet.clermontech.org/tags/lava-jug/) et Clermont'ech s'associent
+pour la première fois afin de vous proposer une soirée exceptionnelle.
+
+## Informations pratiques
+
+Cette soirée est organisée le **jeudi 21 mars** à partir de **18h30** à l'hôtel de région. L'adresse
```
[the remark this answers is not in the document]
Comment (krichprollsch) [reply _posts/2019-03-21-little-big-event-1-typescript.md]:
```suggestion
Cette soirée est organisée le **jeudi 21 mars** à partir de **18h30** dans les locaux du **Bivouac** à l'hôtel de région. L'adresse
```

File: _posts/2019-03-21-little-big-event-1-typescript.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
+---
+layout: blog-post
+categories: [ blog-posts ]
+
+title: Little Big Event#1 Typescript
+---
+
+Le jeudi 21 mars 2019, [Mug in Clermont](https://muginclermont.azurewebsites.net/),
+le [LavaJUG](http://www.lavajug.org/) et Clermont'ech s'associent
+pour la première fois afin de vous proposer une soirée exceptionnelle.
```
Comment (jmaupetit) [inline _posts/2019-03-21-little-big-event-1-typescript.md]: What about le bivouac? :trollface:

File: _posts/2019-03-21-little-big-event-1-typescript.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+## Informations pratiques
+
+Cette soirée est organisée le **jeudi 21 mars** à partir de **18h30** dans les locaux du **Bivouac** à l'hôtel de région. L'adresse
+exacte est : [**L'hôtel de région, 59 Boulevard Léon Jouhaux 63000 Clermont-Ferrand**](https://osm.org/go/0AkOKx_14--)..
```
Comment (jmaupetit) [inline _posts/2019-03-21-little-big-event-1-typescript.md]:
```suggestion
exacte est : [**Hôtel de région, 59 Boulevard Léon Jouhaux 63000 Clermont-Ferrand**](https://osm.org/go/0AkOKx_14--)..
```

File: _posts/2019-03-21-little-big-event-1-typescript.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
+---
+layout: blog-post
+categories: [ blog-posts ]
+
+title: Little Big Event#1 Typescript
```
Comment (jmaupetit) [inline _posts/2019-03-21-little-big-event-1-typescript.md]:
```javascript
'Little Big Event' === '♥'
```

File: _posts/2019-03-21-little-big-event-1-typescript.md (hunk 2 above)
Comment (krichprollsch) [reply _posts/2019-03-21-little-big-event-1-typescript.md]: never heard

File: _posts/2019-03-21-little-big-event-1-typescript.md (hunk 4 above)
Comment (krichprollsch) [reply _posts/2019-03-21-little-big-event-1-typescript.md]:
```
'Little Big Event' === '�'
```

File: _posts/2019-03-21-little-big-event-1-typescript.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+Le jeudi 21 mars 2019, [Mug in Clermont](https://muginclermont.azurewebsites.net/),
+le [LavaJUG](http://www.lavajug.org/) et Clermont'ech s'associent
+pour la première fois afin de vous proposer une soirée exceptionnelle.
+
+## Informations pratiques
+
+Cette soirée est organisée le **jeudi 21 mars** à partir de **18h30** dans les locaux du **Bivouac** à l'hôtel de région. L'adresse
+exacte est : [**Hôtel de région, 59 Boulevard Léon Jouhaux 63000 Clermont-Ferrand**](https://osm.org/go/0AkOKx_14--)..
+
+<iframe width="600" height="400" frameborder="0" scrolling="no" marginheight="0" marginwidth="0" src="https://www.openstreetmap.org/export/embed.html?bbox=3.1138730049133305%2C45.79396934299903%2C3.117982149124146%2C45.79611996814522&amp;layer=mapnik&amp;marker=45.7950446659467%2C3.115927577018738" style="border: none"></iframe>
```
Comment (claudusd) [inline _posts/2019-03-21-little-big-event-1-typescript.md]:
```suggestion
<iframe width="100%" height="400" frameborder="0" scrolling="no" marginheight="0" marginwidth="0" src="https://www.openstreetmap.org/export/embed.html?bbox=3.1138730049133305%2C45.79396934299903%2C3.117982149124146%2C45.79611996814522&amp;layer=mapnik&amp;marker=45.7950446659467%2C3.115927577018738" style="border: none"></iframe>
```

**Pull request merged**